tests/10-EdvClient.spec.js

```diff
@@ -64,6 +64,67 @@ describe('EdvClient', () => {
     docs.length.should.equal(1);
   });
 
+  it('find a document using a multi property query', async () => {
+    const client = await mock.createEdv();
+    client.ensureIndex({attribute: 'content.indexedKey'});
```

Comment:
Is `indexedKey` needed for this test? If not, we should remove it to keep the case simple. And if so -- it's odd that it's named that because the other key is also indexed but it is called `someKey` -- can we rename to avoid confusion?